src/app/_services/helper.service.ts

```diff
@@ -143,4 +149,18 @@ export class HelperService {
     const options = this.generateWeekOptions(new Date(year, 11, 31));
     return of(options.filter((item) => item.key.includes(year)).reverse());
   }
+
+  getFacilityCategories() {
+    return of(Object.entries(FacilityCategory).map(([key, value]) => ({ key, value })));
+  }
+
+  getFacilityTypes(filters: any) {
+    const type = filters[0].value;
```

Comment:
maybe check for filters.length > 0 ?

Reply:
it shouldn't get to this method unless a filter is provided;
we could add a check to be sure